platform/linux-generic/pktio/socket.c

```diff
@@ -697,7 +697,7 @@ static uint32_t _tx_pkt_to_iovec(odp_packet_t pkt,
 				 struct iovec iovecs[MAX_SEGS])
 {
 	uint32_t pkt_len = odp_packet_len(pkt);
-	uint32_t offset = odp_packet_l2_offset(pkt);
+	uint32_t offset = 0;
 	uint32_t iov_count = 0;
 
 	while (offset < pkt_len) {
```

Comment:
that "if" looks like not needed if you set offset to 0.

Reply:
@muvarov Not sure what you're referring to here. There is no if statement is this level of the PR. It's a one-line change to the existing code.

Reply:
Me neither, there is no if statements in this whole function.

Reply:
please skip my comment, it's not relevant to here.